python/foglamp/services/core/routes.py

```diff
@@ -77,6 +78,9 @@ def setup(app):
     # Package Update on demand
     app.router.add_route('PUT', '/foglamp/update', update.update_package)
 
+    # certs upload
+    app.router.add_route('POST', '/foglamp/certificate', certificate_store.upload)
```

Comment:
@MarkRiddoch Please confirm, Are we okay with endpoint?

Reply:
Yes, will also add to admin API document